src/CRM/CivixBundle/Command/AddEntityCommand.php

```diff
@@ -21,7 +23,8 @@ protected function configure() {
     $this
       ->setName('generate:entity')
       ->setDescription('Add a new API/BAO/GenCode entity to a CiviCRM Module-Extension (*EXPERIMENTAL AND INCOMPLETE*)')
-      ->addArgument('<EntityName>', InputArgument::REQUIRED, 'The brief, unique name of the entity")');
+      ->addArgument('<EntityName>', InputArgument::REQUIRED, 'The brief, unique name of the entity")')
+      ->addOption('table-name', NULL, InputOption::VALUE_OPTIONAL, 'The SQL table name. (Default: derived from entity name)');
```

Comment:
How about we add something like: "In most cases it makes sense to let civix derive this automatically from <EntityName>. In some instances (such as when the Entity contains a number) you may prefer to explicitly set a table name." to the usage?

Reply:
@michaelmcandrew It's hard to fit a long explanation into the per-option descriptions; but maybe add a `setHelp()` section? Then it could show a few examples for comparison, like `civix generate:entity ChatLog` and `civix generate:entity SMSLog --table-name=civicrm_sms_log`.